examples/window/window_drag_move.rs

```diff
@@ -0,0 +1,140 @@
+//! This example illustrates drag move and drag resize without window
+//! decorations.
+//!
+//! When window decorations are not present, the user cannot drag the window.
+//! The `start_drag_move()` function will permit the application to make the
+//! window draggable. It does require that the left mouse button was pressed
+//! when it is called.
```

Comment:
When (inconsistent capitalization with other comments)

Reply:
I apologize. I don't quite follow this suggestion. I have rewritten that paragraph. Please see if your comment still applies.